doc/HttpLuaModule.wiki

```diff
@@ -523,12 +523,12 @@ Here is the reason: by design, the global environment has exactly the same lifet
 Generally, use of Lua global variables is a really really bad idea in the context of ngx_lua because
```

Comment:
> a really really bad idea

It seems informal to repeat "really" here. If this PR's scope is to avoid informality, I believe this could be replaced with one of:

```
Generally, use of Lua global variables is [considered harmful] in the context of ngx_lua because:
Generally, use of Lua global variables is [considered a] bad idea in the context of ngx_lua because:
Generally, use of Lua global variables is [considered a very] bad idea in the context of ngx_lua because:
Generally, use of Lua global variables is a [really] bad idea in the context of ngx_lua because:
Generally, use of Lua global variables is a [very] bad idea in the context of ngx_lua because:
Generally, use of Lua global variables is a [really, really] bad idea in the context of ngx_lua because:
```

(note the colon at the end as well)